### ISSUE TITLE
Tensorboard not launching with S3 (minio)

### ISSUE THREAD

**Issue by Lap1n** [polyaxon/polyaxon#530]
### Describe the bug
When using a S3 server for the outputs, I am not able to successfully launch Tensorbard. The error in the UI says : "exist-code(2)-message(None)". When I look at the logs from Minikube, I see the following error : 
```
E0724 13:25:21.871753   19756 pod_workers.go:190] Error syncing pod 340d04ae-38ba-4d36-8f37-2fd4fe2d60d4 ("plx-tensorboard-6feccb08c2044953806a4eed6f1081c9-77599fc56lxgv4_polyaxon(340d04ae-38ba-4d36-8f37-2fd4fe2d60d4)"), skipping: failed to "StartContainer" for "polyaxon-plugin-job" with CrashLoopBackOff: "Back-off 10s restarting failed container=polyaxon-plugin-job pod=plx-tensorboard-6feccb08c2044953806a4eed6f1081c9-77599fc56lxgv4_polyaxon(340d04ae-38ba-4d36-8f37-2fd4fe2d60d4)"
```
Note that I am able to successfully store artifacts on the S3 server. Also, when I don't use a S3 server for the outputs, I am able to launch Tensorboard without errors. 

My helm config looks like this : 
```yaml
deploymentType: minikube

rbac:
  enabled: true

ingress:
  enabled: false

serviceType: NodePort

persistence:
  outputs:
    outputs:
      store: s3
      bucket: s3://polyaxon-outputs
      secret: s3-secret
      secretKey: s3-secret.json
```

Maybe I missed something?

### To Reproduce
Launch Tensorboard using the default config with a Minio S3 server for the outputs

### Expected behavior
To be able to launch Tensorboard with a S3 server
### Environment
Polyaxon version 0.5.3
Minikube
Minio S3 server
Ubuntu 16.04

**Comment by mouradmourafiq** [comment]
This is a "bug" it's seem like Tensorboard can natively handle S3 but not Minio.
I guess some init-containers must be provided by Polyaxon to handle this use-case as well as the Azure case.

**Comment by mouradmourafiq** [comment]
This will be solved automatically when this issue #273 is solved, which should be part of the v0.6 release.
